### DISCUSSION TITLE
Instructions unclear, my PC can now wash my clothes...

### DISCUSSION THREAD

**Discussion by R3tr0PAG** [xCuri0/ReBarUEFI#143] [General]
Just a joke of course.
But in all seriousness, does anyone plan on making like a tutorial? I tried to follow the steps to do this but my brain is like a intel core duo and is hanging on simple things.
Like for example, the tutorial starts somewhere with modding and installing a modded bios... but i don't understand why, because some motherboards already have 4G decoding and rebar support even when the gpu that's installed doesn't support it.

Idk maybe it's just me that overthinks all of this.

Also has anyone attempted this on a gtx 1060 6GB? if so any performance benefits?

**Comment by Apethesis** [comment]
ReBar has known to bring performance benefits, if your GPU lacks ReBar then maybe a custom driver would work? Don't know much about that though.
